### PR TITLE
CHEAP-996 fix: Add AuthorizationController having method for token scope

### PR DESCRIPTION
**Pull request by bholagabbar**

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -76,6 +78,12 @@ class App {
     }
 
     // ADD GETTERS BELOW
+        
+    get Authorization() {
```
Comment (ralph-dev) [inline src/app/App.js:82]: :O isn't it getAuthorization??

File: src/app/controllers/AuthorizationController.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Get the authorization scope (role) for a user in the system through the issued token
+     * @param {string} token - Authorization Token Issued
+     * @returns {Promise<any>} - Role of the user in the system
+     */
+    getTokenScope(token) {
+        let queryString = `
+            query {
+                auth_token_scope(token:"${token}")
```
Comment (ralph-dev) [inline src/app/controllers/AuthorizationController.js]:
```suggestion
                auth_token_scope(token: "${token}")
```

**Review by ralph-dev — APPROVED** [review]
LGTM

File: src/app/App.js (hunk 1 above)
Comment (bholagabbar) [reply src/app/App.js:82]: @ralph-dev no, this is a getter

**Pull request merged**